apps/src/templates/lessonOverview/LessonOverview.jsx

```diff
@@ -87,7 +87,7 @@ class LessonOverview extends Component {
 
         <h2>{i18n.teachingGuide()}</h2>
         {this.props.activities.map(activity => (
-          <Activity activity={activity} key={activity.key} />
+          <Activity activity={activity} key={activity.id} />
```

Comment:
Should Activity and Activity Sections have keys? I know in the past not having a key for objects has made translation harder. It doesn't mean we can't use id here tho.

Reply:
yep: https://codedotorg.atlassian.net/browse/PLAT-407

yeah it's not that important what we use as the react key here. any unique id from the server is a good choice.